modules/net-vlan-attachment/variables.tf

```diff
@@ -22,9 +22,9 @@ variable "admin_enabled" {
 
 variable "bandwidth" {
   # Possible values @ https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_interconnect_attachment#bandwidth
-  description = "The bandwidth assigned to the VLAN attachment (e.g. BPS_10G)."
+  description = "The bandwidth assigned to the VLAN attachment (e.g. BPS_10G). Only applicable for Dedicated Interconnect."

```

Comment:
Just loose comment on this. Maybe it's worth to(comment above) group Partner Interconnect related parameters under one variable, and Dedicated Interconnect related parameters under the other? We could use this to infer `type` for `google_compute_interconnect_attachment` and keep default bandwidth

Reply:
actually that's a good idea, I will do a few changes.

Reply:
Thanks a lot @apichick, LGTM